src/lib/Components/Inbox/Conversations/Message.tsx

```diff
@@ -60,7 +55,7 @@ export default class Message extends React.Component<Props, any> {
   render() {
     return (
       <Container>
-        <Avatar />
+        <Avatar isUser={true} senderName={this.props.message.senderName} />
```

Comment:
Will need to add further code later to show different avatars depending on user / gallery, as well as if gallery, fetching the image :).